src/main/java/frc/robot/subsystems/Vision/VisionIOSim.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout.OriginPosition;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose3d;
+import org.photonvision.PhotonCamera;
+import org.photonvision.simulation.VisionSystemSim;
+
+/** Add your docs here. */
+public class VisionIOSim implements VisionIO {
+  public static final String SIM_VISION_SYSTEM_NAME = "sim vision";
+  public static final String SIM_CAMERA_NAME = "sim camera";
```

Comment:
make sure to have diff cameras have diff names

Reply:
will do once i get sim figured out